lib/grinder_tools.dart

```diff
@@ -175,6 +176,58 @@ class Tests {
   }
 }
 
+/// A wrapper around the `test` package. This class is used to run your unit
+/// tests.
+class Test {
+  final PubApp _test = new PubApp.local('test');
+
+  Test();
+
+  /// Run the tests in the current package. See
+  /// https://pub.dartlang.org/packages/test.
+  ///
+  /// [name] is substring of the name of the test to run. Regular expression
+  /// syntax is supported. [plainName] is a plain-text substring of the name of
+  /// the test to run. [platform] is the platform(s) on which to run the tests.
+  /// Available values are `vm` (default), `dartium`, `content-shell`, `chrome`,
+  /// `phantomjs`, `firefox`, `safari`. [concurrency] controls the number of
+  /// concurrent test suites run (defaults to 4). [pubServe] is the port of a
+  /// pub serve instance serving `test/`.
+  void test({String name, String plainName, List<String> platforms,
+      int concurrency, int pubServe, RunOptions runOptions}) {
+    List<String> args = ['--reporter=expanded'];
+    if (name != null) args.add('--name=${name}');
+    if (plainName != null) args.add('--plain-name=${plainName}');
+    if (platforms != null) args.add('--platform=${platforms.join(',')}');
+    if (concurrency != null) args.add('--concurrency=${concurrency}');
+    if (pubServe != null) args.add('--pub-serve=${pubServe}');
+    _test.run(args, script: 'test', runOptions: runOptions);
+  }
+
+  /// Run the tests in the current package. See
+  /// https://pub.dartlang.org/packages/test.
+  ///
+  /// [name] is substring of the name of the test to run. Regular expression
+  /// syntax is supported. [plainName] is a plain-text substring of the name of
+  /// the test to run. [platform] is the platform(s) on which to run the tests.
+  /// Available values are `vm` (default), `dartium`, `content-shell`, `chrome`,
```

Comment:
Can we just link to the test packages documentation on this instead of inlining it here:

https://github.com/dart-lang/test#platform-selector-syntax

It would also be nice to link to test package for the other docs as well, but those don't seem to have documentation in the README.